.vscode-template/settings.json

```diff
@@ -9,7 +9,6 @@
     "**/*.class": true,
     "**/*.hasTasty": true,
     "**/target/": true,
-    "tests/": true,
```

Comment:
Why ?

Reply:
The commit message: this settings is not only for code search, but also for "ctrl+P" (the quick goto file menu), so having tests there makes its annoying to open test files.